packages/homepage/content/docs/6-templates.md

```diff
@@ -0,0 +1,52 @@
+---
+title: 'Templates'
+authors: ['Saeris']
+description: 'CodeSandbox allows you to create your  custom templates from sandboxes'
+---
+
+## What are Templates?
+
+Templates are a combination of an execution [Environment]() and a set of starter files, similar to what you would find in the output of a framework CLI such as Create React App or Gatsby CLI. CodeSandbox has a number of official Templates, such as React, Vue, Gatsby and others with which you can quickly bootstrap a new project.
```

Comment:
I wonder if we can simplify the explanation a bit. I think we need to drive the point that templates are nothing more than sandboxes with custom metadata to make them more easily accessible. That would make it very easy for people to understand what the concept is.

Reply:
Hmm, maybe. I feel that the docs should provide more detailed technical explanations compared to say, a blog post or a tooltip. I think what you're looking for might be better off in a tooltip.

Reply:
I think it's the correct explanation though, in my mind sandboxes keep the files and execution environment, and templates just contain a sandbox and a color. Explaining that the template keeps the execution environment might lead people to think that only templates keep it, which is not entirely true since sandboxes also keep it.

My way of explaining it would be something like:

"Templates are sandboxes that are easily accessible from your dashboard and the "Create New Sandbox" modal. They are great starting points for a new project. CodeSandbox has a number of official templates, such as React, Vue, Gatsby and others with which you can use to quickly bootstrap a new project."

Reply:
```suggestion
Templates are sandboxes that are easily accessible from your dashboard and the "Create New Sandbox" modal. They are great starting points for new projects. CodeSandbox has a number of official templates, such as React, Vue, Gatsby and others which you can use to quickly bootstrap a new project.
```